nuxeo-distribution/nuxeo-distribution-resources/src/main/resources/templates-tomcat/common-base/nuxeo.defaults

```diff
@@ -15,6 +15,10 @@ nuxeo.db.embeddedDatasources=true
 nuxeo.db.xaMode=false
 nuxeo.db.commonds=jdbc/nuxeo
 
+nuxeo.db.blocking-timeout-millis=10000
+nuxeo.db.idle-timeout-minutes=0
+nuxeo.db.min-pool-size=5
+
```

Comment:
I wonder if these properties should not be in https://github.com/nuxeo/nuxeo/blob/ab88acc3f3ac0f56f08a94a53f6b773c25082aa5/nuxeo-distribution/nuxeo-distribution-resources/src/main/resources/templates-tomcat/nuxeo.defaults with the existing ones:

    nuxeo.db.min-pool-size=5
    nuxeo.db.max-pool-size=100
    nuxeo.db.blocking-timeout-millis=10000

So you would only need to add `nuxeo.db.idle-timeout-minutes=0`.